arangod/ClusterEngine/ClusterEngine.cpp

```diff
@@ -448,14 +420,11 @@ void ClusterEngine::addV8Functions() {
 void ClusterEngine::addRestHandlers(rest::RestHandlerFactory* handlerFactory) {
   ClusterRestHandlers::registerResources(handlerFactory);
 }
-
-Result ClusterEngine::flushWal(bool waitForSync, bool waitForCollector,
-                               bool /*writeShutdownFile*/) {
-  return TRI_ERROR_NO_ERROR;
-}
   
 void ClusterEngine::waitForEstimatorSync(std::chrono::milliseconds maxWaitTime) {
-  // noop
+  // fixes tests by allowing us to reload the cluster selectivity estimates
+  // If test `shell-cluster-collection-selectivity.js` fails consider increasing timeout
+  std::this_thread::sleep_for(std::chrono::milliseconds(2500));
```

Comment:
hopefully there is a better way then to add sleeps in the code

Reply:
Not really, I mean the otheer code was probably also waiting around 1-2s

Reply:
always space for improvement